content/blog/2022-05-17-aws-remotes-in-dvc.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Using AWS Remotes in DVC
+date: 2022-05-17
+description: >
+  We're going to set up an AWS S3 remote in a DVC project.
+descriptionLong: >
+  It can be tricky setting up a remote to handle data versioning in DVC so we're
+  going to go through a tutorial for doing this with AWS.
```

Comment:
Its' true that it can get tricky in some edge scenarios but here we probably mean something like "Setting up remote storage for DVC projects is a popular need" or "There's many ways to setup remote storage for DVC projects so let's look at an example..." or something like that? More positive, less scary.

Also is this about setting up and using remote storage or about data versioning? Or if both what's the connection? I think that if it's beginners content, everything should be very clear and explicit.

Reply:
> is this about setting up and using remote storage or about data versioning?

After reading the content I see its about the former. Probably no need to mention versioning here then? Could be confusing.

Reply:
Let me know what you think about the wording now!

Reply:
Looks good, thanks!